docs/guides/algorithmiq-tem.ipynb

```diff
@@ -227,12 +273,12 @@
    "source": [
     "## Outputs\n",
     "\n",
-    "A Qiskit [PrimitiveResults](/api/qiskit/qiskit.primitives.PrimitiveResult) containing the TEM mitigated result. The result for each PUB is returned as a [PubResult](/api/qiskit/qiskit.primitives.PubResult) containing the following fields:\n",
+    "A Qiskit [PrimitiveResults](https://docs.quantum.ibm.com/api/qiskit/qiskit.primitives.PrimitiveResult) containing the TEM mitigated result. The result for each PUB is returned as a [PubResult](https://docs.quantum.ibm.com/api/qiskit/qiskit.primitives.PubResult) containing the following fields:\n",
```

Comment:
@abbycross, I think this be a relative link, right?

Reply:
Thanks Sanket, I have reverted to relative links.